src/models/quote.js

```diff
@@ -29,211 +56,77 @@ function * makeQuoteQuery (params, config) {
   }
 }
 
-function * getLocalQuote (params, ledgers, config) {
-  const query = yield makeQuoteQuery(params, config)
-  const type = query.source_amount ? 'source' : 'destination'
-  const amount = query.source_amount || query.destination_amount
-  const quote = yield backend.getQuote((yield makeQuoteArgs(query)))
-
-  const validationResult = validator.isValid('Quote', quote)
-  if (!validationResult.valid) {
-    log.error('Quote failed to validate against schema. Quote: ', JSON.stringify(quote))
-    log.error(JSON.stringify(_.omit(validationResult.errors[0], 'stack')))
-    throw new ServerError('Error getting quote from backend')
-  }
-
-  const fixedAmount = type === 'source' ? quote.source_amount
-                                          : quote.destination_amount
-
-  if (fixedAmount !== amount.toString()) {
-    if (!(new BigNumber(fixedAmount).equals(new BigNumber(amount)))) {
-      throw new UnacceptableQuoterAmountError('Backend returned an invalid ' +
-                type + ' amount: ' + fixedAmount + ' (expected: ' + amount + ')')
-    }
+function getLocalTransfer (query) {
+  const amount = query.sourceAmount || query.destinationAmount
+  return {
+    ledger: query.sourceLedger,
+    debits: [{account: query.sourceAccount, amount: amount}],
+    credits: [{account: query.destinationAccount, amount: amount}]
   }
-
-  const sourcePrecisionAndScale = yield precisionCache.get(query.source_ledger)
-  const dstPrecisionAndScale = yield precisionCache.get(query.destination_ledger)
-  const roundedSourceAmount = new BigNumber(quote.source_amount).toFixed(
-    sourcePrecisionAndScale.scale, BigNumber.ROUND_UP)
-
-  const roundedDestinationAmount = new BigNumber(quote.destination_amount).toFixed(
-    dstPrecisionAndScale.scale, BigNumber.ROUND_DOWN)
-
-  validatePrecision(roundedSourceAmount, sourcePrecisionAndScale.precision, 'source')
-  validatePrecision(roundedDestinationAmount, dstPrecisionAndScale.precision, 'destination')
-
-  const roundedQuote = _.assign(_.clone(quote), {
-    source_amount: roundedSourceAmount,
-    destination_amount: roundedDestinationAmount
-  })
-
-  yield validateBalance(query, roundedQuote)
-
-  log.debug('' +
-    roundedQuote.source_amount + ' ' +
-    query.source_ledger + ' => ' +
-    roundedQuote.destination_amount + ' ' +
-    query.destination_ledger)
-
-  return makePaymentTemplate(query, roundedQuote, ledgers)
 }
 
 /**
  * @param {Object} params
- * @param {String} params.source_connector
  * @param {String} params.source_ledger
  * @param {String} params.source_account
  * @param {String} params.source_amount
+ * @param {String} params.source_expiry_duration
  * @param {String} params.destination_ledger
  * @param {String} params.destination_account
  * @param {String} params.destination_amount
+ * @param {String} params.destination_expiry_duration
  * @param {Object} config
- * @returns {Quote[]}
+ * @returns {Transfer}
```

Comment:
If I request a quote with a fixed source amount, how can I tell what the destination amount is?